packages/wallet-service/src/types.ts

```diff
@@ -798,15 +751,4 @@ export interface WalletBalanceValue {
   walletBalanceForTx: TokenBalanceValue[],
 }
 
-/**
- * Alerts should follow the on-call guide for alerting, see
- * https://github.com/HathorNetwork/ops-tools/blob/master/docs/on-call/guide.md#alert-severitypriority
- */
-export enum Severity {
-  CRITICAL = 'critical',
-  MAJOR = 'major',
-  MEDIUM = 'medium',
-  MINOR = 'minor',
-  WARNING = 'warning',
-  INFO = 'info',
-}
+export { Severity, Transaction, TxInput, TxOutput, DecodedOutput };
```

Comment:
Why do you need to export the imported types?

Reply:
We don't, this was done initially because I was only planning to move the types on the next PR

But refactored the code to import from the common project, can you please check again?